cdap-app-fabric/src/test/java/io/cdap/cdap/internal/app/services/http/handlers/SourceControlManagementHttpHandlerTests.java

```diff
@@ -62,9 +74,32 @@ public static void beforeClass() throws Throwable {
 
   @Before
   public void before() {
+    Mockito.reset(getInjector().getInstance(SourceControlManagementService.class));
```

Comment:
it's better to create the instance once in beforeClass (store it in a private static SourceControlManagementService variable). That way it is clear that this is resetting a singleton instance, rather than needing to look before at the initializeAndStart method

Reply:
added the private static SourceControlManagementService variable and moved it into @beforeClass